src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const serviceCenterX = 1400;
+  const serviceCenterY = 500;
```

Comment:
Static constants are usually named uppercase.
```suggestion
  const SERVICE_CENTER_X = 1400;
  const SERVICE_CENTER_Y = 500;
```